examples/backtest.ipynb

```diff
@@ -23,15 +23,33 @@
     "\n",
```

Comment:
<ol><li>could not find -&gt; was not able to find</li><li>Add that you can visualize the residuals not only by timestamp but by any feature</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>